doc/api/fs.md

```diff
@@ -1140,8 +1140,10 @@ changes:
 * `path` {string|Buffer|URL}
 * `mode` {integer}
 
-Synchronously changes the permissions of a file. Returns `undefined`.
-This is the synchronous version of [`fs.chmod()`][].
+Returns `undefined`.
```

Comment:
Is it worth to completely skip these notes? We usually do not state anything if a function returns `undefined`.